packages/investor-foxy/src/utils/buildTxToSign.ts

```diff
@@ -0,0 +1,48 @@
+import { bip32ToAddressNList, ETHSignTx } from '@shapeshiftoss/hdwallet-core'
+import { BIP44Params } from '@shapeshiftoss/types'
+import { numberToHex } from 'web3-utils'
+
+type BuildTxToSignInput = {
+  bip44Params: BIP44Params
+  chainId: number
```

Comment:
pretty sure this will be a caip2, i.e. a string

Reply:
currently, hdwallet-core expects this to be a number